### ISSUE TITLE
Stopping condition for pretraining?

### ISSUE THREAD

**Issue by forresti** [zihangdai/xlnet#37]
Congratulations to the authors on this excellent work.

I'd like to pretrain my own XLNet models. Then, I plan to do some NAS / design space exploration on models similar to XLNet.

My question is: What results on the pretraining set (Wikipedia + BooksCorpus + ...) did you get before you stopped training?
I'm being deliberately vague because I don't know how you measured success on the pretraining set (training loss, test loss, perplexity, ... or what).

Thanks in advance for the help!

**Comment by kimiyoung** [comment]
We measured success by finetuning on downstream tasks.

**Comment by forresti** [comment]
Interesting. How did you decide when to stop pretraining and start finetuning? In other words, how did you decide when your pretraining was "good enough?"

**Comment by stephen-song** [comment]
> Interesting. How did you decide when to stop pretraining and start finetuning? In other words, how did you decide when your pretraining was "good enough?"

also confused about this question, have u got any ideas?
